overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -1913,6 +1913,8 @@ func (s *deviceMgrInstallModeSuite) TestInstallCheckEncryptedStorageSafety(c *C)
 	s.state.Lock()
 	defer s.state.Unlock()
 
+	s.makeMockInstalledPcGadget(c, "", "")
```

Comment:
I was confused by this initially, but it seems this helper should actually be called ...InstalledPcKernelAndGadget